src/System.Runtime.InteropServices.RuntimeInformation/ref/System.Runtime.InteropServices.RuntimeInformation.cs

```diff
@@ -21,8 +21,20 @@ public partial struct OSPlatform : System.IEquatable<System.Runtime.InteropServi
         public static bool operator !=(System.Runtime.InteropServices.OSPlatform left, System.Runtime.InteropServices.OSPlatform right) { return default(bool); }
         public override string ToString() { return default(string); }
     }
+
+    public enum Architecture
+    {
+        x86,
+        x64,
+        ARM
+    }
+
     public static partial class RuntimeInformation
     {
+        public static Architecture GetProcessArchitecture() { return default(Architecture); }
+        public static Architecture GetOSArchitecture() { return default(Architecture); }
+        public static string GetOSDebugName() { return default(string); }
```

Comment:
I just don't see how Debug is better, it's artificial at best. Display achieves the exact same thing, on top of that, it's an opaque string value. That contract implementors like .NET Native, mono and .NET Framework can implement in whatever way makes sense to them.

We should stop trying to protect this so hard, it doesn't add any value.